store/web3.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  VuexAction as Action,
+  VuexMutation as Mutation,
+  VuexModule,
+  Module,
+} from "nuxt-property-decorator"
+import WalletConnectProvider from "@walletconnect/web3-provider"
+import Web3modal from "web3modal"
+import { ethers } from "ethers"
+
+const providerOptions = {
+  walletconnect: {
+    package: WalletConnectProvider,
+    options: { infuraId: "3ce35c3d389a4461bffd073fbf27d23e" },
```

Comment:
So the reason is because of how the provider works on mobile. It needs access to a node before establishing the link.

More on wallet connect:
https://docs.walletconnect.org/quick-start/dapps/web3-provider#setup